docs/using/recipes.md

```diff
@@ -499,3 +499,15 @@ The following recipe demonstrates how to add functionality to read from and writ
 You can now use `pyodbc` and `sqlalchemy` to interact with the database.
 
 Pre-built images are hosted in the [realiserad/jupyter-docker-mssql](https://github.com/Realiserad/jupyter-docker-mssql) repository.
+
+## Add Oracle SQL Instant client, SQL\*Plus and other tools (Version 21.x)
+
+The following recipe demonstrates how to add functionality to connect to a Oracle Database using [Oracle Instant Client](https://www.oracle.com/database/technologies/instant-client.html) in your
+notebook. This recipe installs version 21.+ and it has been validated and manually tested by its author.
```

Comment:
```suggestion
notebook. This recipe installs version 21.+ and it has been validated and manually tested by its author.
```

Let's specify exact version here (because you're installing some exact version).
And, please, remove a comment about validating and manually testing - we have a CI system which runs recipes on schedule and I ask to try the recipe locally or test them myself when adding.